### PR TITLE
Update Firefox.com.xml

### PR DESCRIPTION
**Pull request by Giltyhub**

### CODE REVIEW

File: src/chrome/content/rules/Firefox.com.xml [hunk 1]
```diff
@@ -3,24 +3,24 @@
 
 	These altnames don't exist:
```
Comment (J0WI) [inline src/chrome/content/rules/Firefox.com.xml]: These altnames have been removed.

File: src/chrome/content/rules/Firefox.com.xml [hunk 2]
```diff
@@ -3,24 +3,24 @@
...
 		- scrypt.accounts.firefox.com
-
-	Insecure cookies are set for these hosts:
 		- marketplace.firefox.com
```
Comment (J0WI) [inline src/chrome/content/rules/Firefox.com.xml]: These altname has been removed.

Comment (Giltyhub) [reply src/chrome/content/rules/Firefox.com.xml]: @J0WI removed!

**Comment by alexmuller** [comment]
I think this pull request has broken `detectportal.firefox.com` which has an invalid certificate. My guess is that it doesn't work with TLS because it's designed for captive portal detection.

**Pull request merged**